.github/pull_request_template.md

```diff
@@ -1,21 +1,27 @@
-<!-- Thank you for submitting a pull request! --> 
-<!-- Provide an overview of the PR for maintainers and reviewers. -->
+<!-- Thank you for submitting a pull request! Find more information
+at https://trino.io/development/process.html,
+at https://trinodb.github.io/trino-gateway/development/#contributing
```

Comment:
It's hard to copy these link. Ideally, it shouldn't have a leading `at ` or a following `,`.

Reply:
I was talking to @wendigo in trino PR .. maybe we change these all to markdown outside the comment and at the end of the template.. then the links render properly. Wdyt @ebyhr

Reply:
I can send a follow up PR here or in Trino repo